uiServices/services_test.go

```diff
@@ -0,0 +1,207 @@
+package services
+
+import (
+	"testing"
+	"os"
+	"net/http/httptest"
+	"strings"
+	"github.com/go-chi/chi"
+	"io"
+	"net/http"
+	"github.com/Sirupsen/logrus"
+	"github.com/xtracdev/automated-perf-test/perfTestUtils"
+	"github.com/stretchr/testify/assert"
+)
+
+
+var (
+	reader  io.Reader
+)
+func TestFilePathExist(t *testing.T) {
+	path :=os.Getenv("GOPATH")+"/src/github.com/xtracdev/automated-perf-test/config/"
+	actual := false
+	actual = FilePathExist(path)
```

Comment:
no need to make this function uppercase (i.e. public) if it's in the same package. The test will still be able to access it.